src/js/flows/ingestFiles.ts

```diff
@@ -169,6 +171,13 @@ const trackProgress = (client, gDispatch, clusterId) => {
             gDispatch(space.setIngestProgress(logPostStatusToPercent(status)))
             updateSpaceDetails()
             break
+          case "LogPostResponse":
+            console.log(status)
+            updateSpaceDetails()
+            ;(status.warnings || []).forEach((warning) => {
```

Comment:
I think this semi-colon isn't doing anything. Also, would recommend using lodash's `forEach` for this instead

Reply:
Yeah, prettier added that in. I guess when a line starts with parens, it adds the semi-colon.